sdk/storage/storage-queue/test/queueclient.spec.ts

```diff
@@ -208,3 +197,50 @@ describe("QueueClient", () => {
     assert.strictEqual(tracer.getActiveSpans().length, 0, "All spans should have had end called");
   });
 });
+
+describe("QueueClient - Verify Name Properties", () => {
+  let queueName = "queueName";
+  let accountName = "myAccount";
+
+  function verifyNameProperties(url: string, accountName: string, queueName: string) {
+    const newClient = new QueueClient(url);
+    assert.equal(newClient.name, queueName, "Queue name is not the same as the one provided.");
+    assert.equal(
+      newClient.accountName,
+      accountName,
+      "Account name is not the same as the one provided."
+    );
+  }
+
+  it("verify accountName and queueName passed to the client - Endpoint from the portal", async () => {
+    verifyNameProperties(
+      `https://${accountName}.queue.core.windows.net/` + queueName,
+      accountName,
+      queueName
+    );
+  });
+
+  it("verify accountName and queueName passed to the client - IPv4 style Endpoint", async () => {
+    verifyNameProperties(
+      `https://192.0.0.10:1900/${accountName}/${queueName}`,
+      accountName,
+      queueName
+    );
+  });
+
+  it("verify accountName and queueName passed to the client - IPv6 style Endpoint", async () => {
```

Comment:
Test says ipv6 style, but the host in the URL is an IPv4 address.

Reply:
Thanks! Good catch :)